test/foundry-tests/TickMath.t.sol

```diff
@@ -0,0 +1,212 @@
+pragma solidity ^0.8.20;
+
+import "forge-std/console.sol";
+import {Test} from "forge-std/Test.sol";
+import {Vm} from "forge-std/Vm.sol";
+import {TickMathTest} from "../../contracts/test/TickMathTest.sol";
+import {TickMath} from "../../contracts/libraries/TickMath.sol";
+
+contract TickMathTestTest is Test {
+    int24 constant MIN_TICK = -887272;
+    int24 constant MAX_TICK = -MIN_TICK;
+
+    uint160 constant MIN_SQRT_RATIO = 4295128739;
+    uint160 constant MAX_SQRT_RATIO = 1461446703485210103287273052203988822378723970342;
+
+    uint160 constant SQRT_RATIO_1_1 = 79228162514264337593543950336;
+
+    uint256 constant ONE_PIP = 1e6;
+
+    uint160[] getSqrtRatioAtTickFuzzResults;
+    int24[] getTickAtSqrtRatioFuzzResults;
+
+    TickMathTest tickMath;
+
+    function setUp() public {
+        tickMath = new TickMathTest();
+        delete getSqrtRatioAtTickFuzzResults;
```

Comment:
shouldn't need this, was this there for a particular reason?

Reply:
for good measure, in case another test wants to use it?? just felt cleaner...